bin/travis/before_script.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+"$(dirname ${BASH_SOURCE[0]})/../../node_modules/.bin/sequelize" \
+  --config /etc/blocktogether/sequelize.json db:migrate
+"$(dirname ${BASH_SOURCE[0]})/../../run-dev.sh" \
+  > "$(dirname ${BASH_SOURCE[0]})/../../bt.log" 2>&1 &
+sleep 10
```

Comment:
I prefer a while loop for this type of thing, e.g.:

`until curl localhost:3000 ; do echo "Waiting for service..."; sleep 1; done`